app/policies/application_policy.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+# Default Policies
+class ApplicationPolicy
+  attr_reader :user, :record
+
+  def initialize(user, record)
+    @user = user
+    @record = record
```

Comment:
So rename it to `resource` and remove most (all?) initializers in the inheriting policies (which all use `resource` then)? Would also work.